client/app/lib/list-ctrl.js

```diff
@@ -60,6 +60,8 @@ export default class ListCtrl {
       orderByReverse: this.pageOrderReverse,
     });
 
+    this.setCurrentPage = page => this.paginator.setPage(page);
```

Comment:
In `outdated-queries/index.js` you call `$applyAsync` after calling `setPage`. It's not needed here?

Reply:
Hm, good catch. It works here without `$applyAsync` (I tested all pages except of outdated queries - because I don't know how to fill it with some items), and everywhere I had to add `$applyAsync`. It's strange that it works here

Reply:
It works without `$applyAsync`, but it shouldn't. Probably it's some side-effect, so I added `$applyAsync` to ensure that it will not accidentally break